### PR TITLE
Dev/saars/console example

### PR DESCRIPTION
**Pull request by xiaomi7732**
* Add an example for Console Application Insights to enable AI.K8s. Verify worked.
* Replace NuGet package deprecated licenseUrl property.
* Fix some typos in the code.
* Bump up the version number.

### CODE REVIEW

**Comment by xiaomi7732** [comment]
As an exmaple for #177.

**Comment by karolz-ms** [comment]
Sorry, I saw this and then is somehow fell through the cracks 😳 Looking now...

File: examples/BasicConsoleApp/README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# Enable Application Insights for Kubernetes in .NET Core Console Application
+
+When running a Console Application inside a Kubernetes managed container, and if you have chosen Application Insights as your telemetry solution, you might want to enrich the telemetry with Kubernetes related information. This is an example of how to do it.
+
+_Note: This is an example that does not follow all best practices, including security-related best practices. E.g. Application Insights instrumentation key is not adequately protected (it should be deployed as a secret)._
+
+## Prerequisite
```
Comment (karolz-ms) [inline examples/BasicConsoleApp/README.md]: prerequisites (plural)

File: src/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (karolz-ms) [inline src/.vscode/launch.json]: VS Code is not required to run the sample so I would keep it minimal and not include this

File: src/.vscode/tasks.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
+{
```
Comment (karolz-ms) [inline src/.vscode/tasks.json]: Remove?

**Comment by xiaomi7732** [comment]
This example should address #173 too.

File: examples/BasicConsoleApp/README.md (hunk 1 above)
Comment (xiaomi7732) [reply examples/BasicConsoleApp/README.md]: Sure.

File: src/.vscode/launch.json (hunk 2 above)
Comment (xiaomi7732) [reply src/.vscode/launch.json]: These are under src/ and are actually used to build the production code. They are relocated from the root of the repo.
The reason behind is to reduce the interference to the prod code from the examples.

I would not say this is the best place to put this change but at the same time, I don't think it worth another PR just for the relocate of these jsons.

**Comment by xiaomi7732** [comment]
@karolz-ms  thanks for the review!

**Pull request merged**